core/admin/mailu/manage.py

```diff
@@ -42,22 +42,44 @@ def advertise():
 @click.argument('localpart')
 @click.argument('domain_name')
 @click.argument('password')
+@click.option('-m', '--mode')
 @flask_cli.with_appcontext
-def admin(localpart, domain_name, password):
+def admin(localpart, domain_name, password, mode = 'create'):
```

Comment:
I know we do not fully comply with pep8, but there should be no spaces around that '='.